model/accounts_mgmt/v1/billing_models_resource.model

```diff
@@ -0,0 +1,16 @@
+resource BillingModels {
+    // Retrieves a list of BillingModels.
+	method List {
+		// Index of the requested page, where one corresponds to the first page.
+		in out Page Integer = 1
+
+		// Maximum number of items that will be contained in the returned page.
+		in out Size Integer = 100
+
+		// Total number of items of the collection, regardless of the size of the page.
+		out Total Integer
+
+		// Retrieved list of Capabilities.
```

Comment:
@tirthct probably copy and paste issue. This returns a list of Billing Models

Reply:
Fixed